docs/unit-converter-requirements.md

```diff
@@ -0,0 +1,17 @@
+# Make a unit converter
+
+## Interface
+Get creative.
```

Comment:
This is nitpicking on language but I don't think this line is needed ("get creative").